maps/away/away_site/stranded/stranded.dm

```diff
@@ -0,0 +1,56 @@
+/datum/map_template/ruin/away_site/stranded
+	name = "grove planet with escape pods"
+	description = "Automated emergency distress signals are being emitted from this grove planet. They appear to be coming from two escape pods currently landed there. Scans also indicate irregular bluespace activity within the exosphere."
+	suffixes = list("away_site/stranded/stranded.dmm")
+	sectors = list(SECTOR_BADLANDS)
+	spawn_weight = 1
+	spawn_cost = 2
+	id = "stranded"
+
+/singleton/submap_archetype/stranded
+	map = "grove planet with escape pods"
+	descriptor = "Automated emergency distress signals are being emitted from this grove planet. They appear to be coming from two escape pods currently landed there. Scans also indicate irregular bluespace activity within the exosphere."
+
+/obj/effect/overmap/visitable/sector/stranded
+	name = "grove planet with escape pods"
+	desc = "Automated emergency distress signals are being emitted from this grove planet. They appear to be coming from two escape pods currently landed there. Scans also indicate irregular bluespace activity within the exosphere."
+
+/area/stranded
+	name="survivor planet"
+	icon_state = "outpost_mine_main"
+	requires_power = FALSE
+
+/area/stranded/forest
+	name="survivor forest"
+	icon_state = "outpost_mine_main"
+	requires_power = FALSE
+
+/area/stranded/forest/play_ambience(var/mob/living/L)
+	..()
+	if(L && L.client && (L.client.prefs.sfx_toggles & ASFX_AMBIENCE) && !L.ear_deaf)
+		L.playsound_to(get_turf(L),sound('sound/ambience/spooky_jungle.ogg', repeat = 1, wait = 0, volume = 15, channel = 1))
+
+/area/stranded/forest/Exited(var/mob/living/L, atom/newarea)
+	if(!istype(newarea, /area/stranded/forest))
+		L << sound(null, channel = 1)
```

Comment:
Replace all these `<< sound` with `sound_to(L, sound(...))`

Reply:
I don't think that comment has been resolved